src/tests/Loader/classloader/regressions/GitHub_74792/GitHub_74792.cs

```diff
@@ -0,0 +1,35 @@
+using System;
```

Comment:
Needs banner.

Reply:
I have added the banner and I moved the test under the StaticVirtualMethods folder.